hash/ibm5150.xml

```diff
@@ -5947,6 +5947,22 @@ has been replaced with an all-zero block. -->
 		</part>
 	</software>
 
+	<software name="msdos32a">
+		<description>MS-DOS (Version 3.2) (Alt.)</description>
```

Comment:
Changing `Alt.` to `alt` would be consistent with the rest of this software list.